.github/workflows/elixir.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Compile (warnings as errors)
         run: mix compile --force --warnings-as-errors
       - name: Run tests
-        run: mix test --cover --no-compile
+        run: mix test --cover
```

Comment:
I believe it was there because `mix compile` is there in the previous step. However, `mix compile` runs in the default environment, which is `dev`. `mix test` runs in the `test` environment, which would normally compile a separate version of the app for testing: https://elixir-lang.org/getting-started/mix-otp/introduction-to-mix.html#environments. I think with `--no-compile` the test action was picking up an outdated version of the compiled code.

Reply:
Ok cool